plugins/woocommerce-blocks/assets/js/blocks/product-collection/block.json

```diff
@@ -37,13 +37,18 @@
 		},
 		"queryContextIncludes": {
 			"type": "array"
+		},
+		"location": {
+			"type": "object"
 		}
 	},
+	"usesContext": [ "postId", "templateSlug" ],
 	"providesContext": {
 		"id": "id",
 		"queryId": "queryId",
 		"query": "query",
 		"displayLayout": "displayLayout",
+		"location": "location",
```

Comment:
We'd need to also add this under the ProductTemplate's `usesContext` key

Reply:
> We'd need to also add this under the ProductTemplate's usesContext key

I'm not sure that's needed. Are we gonna consume it from context at all? Right now it's consumed in Product Template through mechanism from https://github.com/woocommerce/woocommerce/pull/44150 that takes location from Product Collection `attributes`. Let me know if there's a need to expose it or you agree?

Reply:
Yes, #44150 is responsible for handling everything related to the editor/preview mode. To grab the context from the attributes in the SSR, it needs to be passed through the Product Collection context into the Product Template's context. This will enable it to be used in SSR. Does that make sense?